src/pygeoprocessing/geoprocessing.py

```diff
@@ -2495,37 +2487,48 @@ def convolve_2d(
             kernel_block[numpy.isclose(kernel_block, kernel_nodata)] = 0.0
         kernel_sum += numpy.sum(kernel_block)
 
+    work_queue = queue.Queue()
+    signal_offset_list = list(iterblocks(s_path_band, offset_only=True))
+    kernel_offset_list = list(iterblocks(k_path_band, offset_only=True))
+    n_blocks = len(signal_offset_list) * len(kernel_offset_list)
+
+    LOGGER.debug(f'start fill work queue thread')
```

Comment:
no leading `f` necessary here.

Reply:
Removed in 79409027d7d280b665fd2812aa4437350acfa59b